### PR TITLE
[fix] configurable concurrency

### PR DESCRIPTION
**Pull request by kuannie1**

### CODE REVIEW

File: cmd/serve-config.go [hunk 1]
```diff
@@ -97,6 +107,15 @@ func serveConfigRun(cmd *cobra.Command, args []string) error {
 	awsSession, err := session.NewSessionWithOptions(
 		session.Options{
 			SharedConfigState: session.SharedConfigEnable,
+			Config: aws.Config{
+				Retryer: &client.DefaultRetryer{
```
Comment (kuannie1) [inline cmd/serve-config.go:111]: Unsure if this means we don't have to configure a retryer for IAM sessions

File: cmd/serve-config.go [hunk 2]
```diff
@@ -29,9 +32,16 @@ type AWSEnvironment struct {
...
+var mappingConcurrencyLimit int
+var rolesConcurrencyLimit int
+var awsSessionRetries int
+
 func init() {
 	rootCmd.AddCommand(serveConfigCmd)
 	serveConfigCmd.Flags().IntVar(&webServerPort, "web-server-port", 8080, "port to host the aws config website")
+	serveConfigCmd.Flags().IntVar(&mappingConcurrencyLimit, "mapping-concurrency-limit", 1, "Number of parallel processes for adding to the AWS Org's config mapping")
+	serveConfigCmd.Flags().IntVar(&rolesConcurrencyLimit, "aws-roles-concurrency-limit", 1, "Number of parallel AWS list-roles and list-tags processes")
+	serveConfigCmd.Flags().IntVar(&awsSessionRetries, "aws-retries", 1, "Number of times an AWS svc retries an operation")
```
Comment (edulop91) [inline cmd/serve-config.go:44]:
```suggestion
	serveConfigCmd.Flags().IntVar(&awsSessionRetries, "aws-retries", 5, "Number of times an AWS svc retries an operation")
```

File: cmd/serve-config.go [hunk 3]
```diff
@@ -97,6 +107,15 @@ func serveConfigRun(cmd *cobra.Command, args []string) error {
 	awsSession, err := session.NewSessionWithOptions(
 		session.Options{
 			SharedConfigState: session.SharedConfigEnable,
+			Config: aws.Config{
+				Retryer: &client.DefaultRetryer{
+					NumMaxRetries:    awsSessionRetries,
+					MinRetryDelay:    time.Millisecond,
+					MinThrottleDelay: time.Millisecond,
+					MaxThrottleDelay: time.Second,
+					MaxRetryDelay:    time.Second,
```
Comment (edulop91) [inline cmd/serve-config.go:116]:
```suggestion
					MaxRetryDelay:   10*  time.Second,
```

File: cmd/serve-config.go [hunk 4]
```diff
@@ -97,6 +107,15 @@ func serveConfigRun(cmd *cobra.Command, args []string) error {
 	awsSession, err := session.NewSessionWithOptions(
 		session.Options{
 			SharedConfigState: session.SharedConfigEnable,
+			Config: aws.Config{
+				Retryer: &client.DefaultRetryer{
+					NumMaxRetries:    awsSessionRetries,
+					MinRetryDelay:    time.Millisecond,
+					MinThrottleDelay: time.Millisecond,
+					MaxThrottleDelay: time.Second,
```
Comment (edulop91) [inline cmd/serve-config.go:115]:
```suggestion
					MaxThrottleDelay: 10 * time.Second,
```

File: cmd/serve-config.go (hunk 1 above)
Comment (edulop91) [reply cmd/serve-config.go:111]: This feels like the right place but I also have the same doubt as you. Not sure

**Comment by kuannie1** [comment]
@edulop91 The listroles function takes a really long time here... I'm not sure why.

**Pull request merged**